include/js/general.js

```diff
@@ -3157,7 +3157,13 @@ function ActivityReminderRemovePopupDOM(id) {
 
 /* ActivityReminder Customization: Pool Callback */
 var ActivityReminder_regcallback_timer;
-var ActivityReminder_Deactivated = false;
+var ActivityReminder_Deactivated;
```

Comment:
leave the assignment to "false" because the call to get the correct value is ajax and takes some time so we don't want to have the code executing without a value before the variable gets assigned.